data-raw/watershed/stanislaus/stainislaus_river.rmd

```diff
@@ -23,22 +23,65 @@ None Planned
 
-```{r,include=FALSE}
- stanislaus_raw <- read_excel('data-raw/mark_gard_data/IFIMWUA.xlsx', range = "I4:N60", sheet = 'Stanislaus',
-                              col_names = c('flow_cfs', 'FR_fry_wua', 'ST_fry_wua', 'FR_juv_wua', 'ST_juv_wua', 'FR_spawn_wua'))
- stanislaus_raw
+```{r,echo=FALSE}
+# Rearing data for Steelhead and Fall Run Chinook 
+reach_km <- c(6.4, 12.4, 22, 55.6)
+names(reach_km) <- 1:4
+total <- sum(reach_km)
+
+stan_rearing_1 <- read_excel('data-raw/mark_gard_data/IFIMWUA.xlsx', 
+                              range = "A4:E33", sheet = 'Stanislaus', 
+                              col_names = c('flow_cfs', 'FR_fry_wua',
+                              'ST_fry_wua', 'FR_juv_wua', 'ST_juv_wua')) %>%
+  select(flow_cfs, FR_fry_wua, ST_fry_wua, FR_juv_wua, ST_juv_wua) %>%
+  mutate(km = reach_km[1])
+
+stan_rearing_2 <- read_excel('data-raw/mark_gard_data/IFIMWUA.xlsx', 
+                              range = "A38:E67", sheet = 'Stanislaus', 
+                              col_names = c('flow_cfs', 'FR_fry_wua',
+                              'ST_fry_wua', 'FR_juv_wua', 'ST_juv_wua')) %>%
+  select(flow_cfs, FR_fry_wua, ST_fry_wua, FR_juv_wua, ST_juv_wua) %>%
+  mutate(km = reach_km[2])
+
+stan_rearing_3 <- read_excel('data-raw/mark_gard_data/IFIMWUA.xlsx', 
+                              range = "A38:E67", sheet = 'Stanislaus', 
+                              col_names = c('flow_cfs', 'FR_fry_wua',
+                              'ST_fry_wua', 'FR_juv_wua', 'ST_juv_wua')) %>%
+  select(flow_cfs, FR_fry_wua, ST_fry_wua, FR_juv_wua, ST_juv_wua) %>%
+  mutate(km = reach_km[3])
+
+stan_rearing_4 <- read_excel('data-raw/mark_gard_data/IFIMWUA.xlsx', 
+                              range = "A38:E67", sheet = 'Stanislaus', 
+                              col_names = c('flow_cfs', 'FR_fry_wua',
+                              'ST_fry_wua', 'FR_juv_wua', 'ST_juv_wua')) %>%
+  select(flow_cfs, FR_fry_wua, ST_fry_wua, FR_juv_wua, ST_juv_wua) %>%
+  mutate(km = reach_km[4])
+
+# Scale rearing total data by length 
+stanislaus_raw <- stan_rearing_1 %>% 
+  bind_rows(stan_rearing_2, stan_rearing_3, stan_rearing_4) %>% 
+  gather(lifestage, sq_ft, -flow_cfs, -km) %>%
+  group_by(lifestage, flow_cfs) %>%
+  summarise(wua = sum((((sq_ft/km) * .3048) * (km/total)), na.rm = TRUE)) %>%
+  spread(lifestage, wua) 
+
+
+# Spawning data for Fall Run 
+stan_spawn <- read.csv('data-raw/watershed/stanislaus/data/stanislaus_spawning_data.csv', 
+                       skip = 1) %>% 
+  select(flow_cfs, FR_total_spawn_WUA) %>%
+  rename(FR_spawn_wua = FR_total_spawn_WUA)
 
 ```
 
 
 ```{r, echo=FALSE}
-stanislaus_river_instream <- stanislaus_raw %>% 
+stanislaus_river_instream <- stanislaus_raw %>% bind_rows(stan_spawn) %>%
    mutate(watershed = 'Stanislaus River') %>% 
    select(flow_cfs, FR_spawn_wua, FR_fry_wua, FR_juv_wua, ST_fry_wua, ST_juv_wua, watershed)
  
```

Comment:
It looks like you added new spawning data? it doesn't seem like you ran the `use_data` command to update `stanisalus_river_instream`. please do if the data has changed. there should be a stanislaus_river_instream.rda file added to this PR
also, change line 89 to this: 
`# usethis::use_data(stanislaus_river_instream, overwrite = TRUE)`
`use_data` has been moved from devtools

Reply:
Should be fixed now.